bin/bench

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+import os, re, subprocess, signal, sys, tempfile, time
+
+def fail():
+  print "usage: " + sys.argv[0] + " <suite>"
+  print ""
+  print "Supported bench suites:"
+  print "  * All: reproduces docs/performance.md"
+  print "  * CI: Rsc-specific subset of All that is run for every commit in CI"
```

Comment:
For every push to github?

Reply:
I think it is useful to have full performance information about git history. For the current bench suite and bench box, we can do ~25-30 commits a day, which should be enough for the time being.

Reply:
I wasn’t very clear here, I was suggesting a text change as we can’t actually make Travis-CI run on every commit.

Reply:
We can't Travis to run benchmarks. This will be done by a dedicated service that runs on our benchbox, implemented within https://github.com/twitter/reasonable-scala/issues/3.